README.md

```diff
@@ -75,15 +92,50 @@ class ParserTests {
     void fuzzTest(@NotNull String input) {
         assertEquals(input, SomeScheme.decode(SomeScheme.encode(input)));
     }
+	
+    @FuzzTest
+    void fuzzTest(@NotNull @WithUtf8Length(min=10, max=200) String input, @InRange(min=-10, max=10) int n) {
```

Comment:
Doesn't this function need to have a different name as one function `fuzzTest` already exists?